### PR TITLE
[4913][ADD] res_users_chatter

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[4913](https://www.quartile.co/web#id=4913&cids=3&menu_id=505&action=1457&model=project.task&view_type=form)

### CODE REVIEW

File: purchase_budget_line/static/description/index.html [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```
Comment (kanda999) [inline purchase_budget_line/static/description/index.html]: This change should be separated into a separate PR.

**Comment by AungKoKoLin1997** [comment]
This PR will be closed and create in axls-oca after related OCA PR  https://github.com/OCA/server-ux/pull/965 is approved.
